src/Sarif.FunctionalTests/v2/ConverterTestData/FxCop/Readme.md

```diff
@@ -0,0 +1,15 @@
+Notes on producing test examples.
+
+This output is an FxCop run against a debug build of SARIF-SDK 
+
+0. Enlist in the SARIF-SDK. You will update the test files in this version.
+1. Git clone into a second version of the tree, e.g.:
```

Comment:
>version of the t [](start = 27, length = 16)

Again, consider: "version of the tree" => "copy of the repo".

Nit, but "version" means something specific in a VCS.